esphome/components/http_request/http_request_idf.cpp

```diff
@@ -19,14 +19,47 @@ namespace http_request {
 
 static const char *const TAG = "http_request.idf";
 
+struct UserData {
+  const std::set<std::string> &collect_header_names;
+  std::map<std::string, std::list<std::string>> response_headers;
+};
+
 void HttpRequestIDF::dump_config() {
   HttpRequestComponent::dump_config();
   ESP_LOGCONFIG(TAG, "  Buffer Size RX: %u", this->buffer_size_rx_);
   ESP_LOGCONFIG(TAG, "  Buffer Size TX: %u", this->buffer_size_tx_);
 }
 
+esp_err_t HttpRequestIDF::http_event_handler(esp_http_client_event_t *evt) {
+  ESP_LOGD(TAG, "Entered http_event_handler");
```

Comment:
If you really want to keep the log, it should be made verbose; this is being logged very often otherwise
```suggestion
  ESP_LOGV(TAG, "Entered http_event_handler");
```

Reply:
I removed it.